src/app/pages/collections/search/landing-page.jsx

```diff
@@ -180,8 +180,31 @@ function ChplLandingPage() {
               <ChplFilterSearchTerm />
               <ChplFilterQuickFilters />
             </Box>
-            <Box pb={4}>
-              <Typography className={classes.subHeaders} align="left" variant="h2" gutterBottom>
+            { editionlessIsOn ? (
+              <>
+                <Box pb={4}>
+                  <Typography className={classes.subHeaders} align="left" component="h2" variant="h2" gutterBottom>
+                    Feature Topic
+                  </Typography>
+                </Box>
+                <Card className={classes.collectionsCards}>
+                  <CardContent>
+                    <Box display="flex" flexDirection="row" gridGap={8}>
+                      <Box className={classes.collectionsCardText}>
+                        <Typography align="left" component="h3" variant="h4" color="primary">
+                          <strong>Discontinuing Year-Themed Editions for Health IT Certification Criteria</strong>
+                        </Typography>
+                        <Typography>
+                          To simplify the Certification Program and support more modular and extensible future updates, the HTI-1 final rule discontinues year-themed editions of certification criteria. This change also supports broader use of certification criteria and standards adopted by ONC for other federal agencies and programs.
+                        </Typography>
+                      </Box>
+                    </Box>
+                  </CardContent>
+                </Card>
+              </>
+            ) : null}
```

Comment:
Changing it to a conditional makes this something like:
```suggestion
            )}
```